src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      return this.result + num;
+    },
+    subtract(num) {
+      return this.result - num;
+    },
```

Comment:
add 1 empty line between methods. Fix all cases
```suggestion

    add(num) {
      return this.result + num;
    },
    
    subtract(num) {
      return this.result - num;
    },
```